app/org/sagebionetworks/bridge/dynamodb/DynamoInitializer.java

```diff
@@ -490,18 +488,6 @@ static void compareSecondaryIndices(String tableName, List<?> indices1, List<?>
                             " is changing the projection.");
                 }
 
-                if (!globalIndex1.getProvisionedThroughput().getReadCapacityUnits().equals(
-                        globalIndex2.getProvisionedThroughput().getReadCapacityUnits())) {
-                    throw new BridgeInitializationException("Table " + tableName + " global index " + indexName +
-                            " is changing the provisioned read capacity.");
-                }
```

Comment:
Let's take this a step further and wipe out compareSecondaryIndices() altogether. Otherwise, this will prevent us from creating, deleting, or modifying indices in the future.